docs/user/tutorials/installing.md

```diff
@@ -1,11 +1,49 @@
 # Installing Hypper
 
-## From source (Linux, macOS)
+This guide shows how to install the Hypper CLI. Hypper can be installed either from source, or from pre-built binary releases.
+
+
+## From the Binary Releases (Linux, macOS, windows)
```

Comment:
Nitpicking:

```suggestion
## From Binary Releases (Linux, macOS, Windows)
```

Reply:
`s/Binary Releases/binary releases/` while we are at it :)